schunk_gripper_driver/schunk_gripper_driver/driver.py

```diff
@@ -89,10 +93,34 @@ def __init__(self, node_name: str, **kwargs):
         self.show_configuration_srv = self.create_service(
             ShowConfiguration, "~/show_configuration", self._show_configuration_cb
         )
+        self.add_on_set_parameters_callback(self._param_cb)
 
         # For concurrently running publishers
         self.callback_group = MutuallyExclusiveCallbackGroup()
 
+    def _param_cb(self, params):
+        for p in params:
+            if p.name == "log_level":
+                if p.value not in [
```

Comment:
I would suggest creating a variable, such as `valid_levels = [..]` and then use that for the check and also for the feedback in the error message.